tools/autograd/derivatives.yaml

```diff
@@ -713,6 +713,22 @@
 - name: special_xlog1py.other_scalar(Tensor self, Scalar other) -> Tensor
   self: grad * log1p(other.toDouble())
 
+- name: special_betainc(Tensor self, Tensor a, Tensor b) -> Tensor
+  self: not_implemented("special_betainc")
+  a: not_implemented("special_betainc")
+  b: not_implemented("special_betainc")
+
+- name: special_betainc.a_scalar(Tensor self, Scalar a, Tensor b) -> Tensor
+  self: not_implemented("special_betainc")
+  b: not_implemented("special_betainc")
+
+- name: special_betainc.b_scalar(Tensor self, Tensor a, Scalar b) -> Tensor
+  self: not_implemented("special_betainc")
+  a: not_implemented("special_betainc")
+
+- name: special_betainc.ab_scalar(Tensor self, Scalar a, Scalar b) -> Tensor
+  self: grad
```

Comment:
Stray?

Reply:
I forgot to put it as `not_implemented`. Fixing now!